docs/migrate-to-managed-asm/service-entry.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START servicemesh_migrate_to_managed_asm_service_entry_serviceentry_external_service_http]
+# With this ServiceEntry, we're instructing the mesh to
+# direct HTTP requests (from the mesh) for v2.example.com to the specified IP address.
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: external-service-http
+spec:
+  hosts:
+  - v2.example.com
+  location: MESH_EXTERNAL
+  ports:
+  - number: 80
+    name: http
+  resolution: STATIC
+  endpoints:
+  - address: 1.2.3.4 # Replace this IP address with the IP address of the ASM ingress gateway.
```

Comment:
did you want to leave this as `1.2.3.4`, or maybe a place holder? 
`YOUR_IP_ADDRESS`, and maybe a command that can replace it through terminal?